js/framework/AnalyticsConstants.js

```diff
@@ -324,6 +363,54 @@ if (!OO.Analytics.EVENT_DATA)
     this.totalStreamDuration = checkVideoStreamPositionChangedData(totalStreamDuration, "totalStreamDuration", "number");
   }
 
+  /**
+   * @public
+   * @class Analytics.EVENT_DATA#AdPodStartedData
+   * @classdesc Contain information about how many ads are in the ad pod.
+   * @property {number} numberOfAds
```

Comment:
I know it looks redundant, but please fill in the description for the variable.  When we use the JSDoc parser it will generate better official documentation.